.github/workflows/testing.yml

```diff
@@ -125,6 +130,7 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
+          persist-credentials: false
```

Comment:
To help in my understanding: how does a later `git fetch` work if the credentials aren't there any more?

Reply:
I think it works because we only need public auth for the following fetch